### DISCUSSION TITLE
v2 to v3 How to upgrade

### DISCUSSION THREAD

**Discussion by Nokecy** [elsa-workflows/elsa-core#3096] [Q&A]
Are there any breaking updates ？

**Comment by sfmskywalker** [comment]
V3 is a complete rewrite and it's still under heavy development, so it's too early to offer a migration guide at this point.
Once we are close to a preview release, we can hopefully look into ways to migrate V2 workflows to V3.

**Reply by benbhall** [reply]
It's more than workflow migration though if we have customised it with new activities, fields etc. We're exploring options at the moment. I love Elsa but perhaps adopting it at v2 is not such a good idea if we need to customise it quite a bit as we could get trapped on v2 that will likely stop getting updates?
